src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/operations/content_key_policy.py

```diff
@@ -89,8 +91,11 @@ def _generate_content_key_policy_object(client, resource_group_name, account_nam
     if open_restriction and valid_token_restriction:
         raise CLIError('You should only use one restriction type.')
 
-    if clear_key_configuration:
+    if clear_key_configuration and not widevine_template:
         configuration = ContentKeyPolicyClearKeyConfiguration()
+    elif widevine_template:
```

Comment:
leave an empty line between

Reply:
I don't get this. Why would I need to leave an empty line there?